qiskit_metal/qlibrary/core/qroute.py

```diff
@@ -296,6 +296,13 @@ def set_lead_extension(self, name: str) -> QRoutePoint:
                 lead.go_right(length)
             elif turn in ("straight", "D", "S"):
                 lead.go_straight(length)
+            elif turn in ("right45", "R45"):
+                lead.go_right45(length)
+            elif turn in ("left45", "L45"):
+                lead.go_left45(length)
+            elif ('A' or 'angle') in turn:
```

Comment:
This looks right, but let's think generalizations. Is there any reason we should introduce the A factor instead of generalizing the L** (left**) and R** (right**)?

Reply:
I don't think so in particular. A just provides a way of being able to provide an angle in any direction +/- where as L and R will enforce the direction. Perhaps L** R** is more consistent with the previous

Reply:
Nice L)

Reply:
Alright, I extended this case coverage and created #574. (Separated the two, to add further work such as test notebooks, unit test and other documentation)